docs-src/concepts/what-is-the-temporal-cli.md

```diff
@@ -28,3 +26,187 @@ Workflow, show Workflow History, and Signal Workflow).
+2. Download the binary.
+3. Extract the downloaded archive.
+4. Add the `temporal` binary to your system PATH.
+
+## Starting the Temporal Server
+
+Run the command provided below to start the Temporal Server.
+This will automatically start the Web UI.
+
+```bash
+temporal server start-dev
+```
+
+At this point you should have a server running on `localhost:7233` and a web interface at <http://localhost:8233>.
+A default Namespace has also been created.
+
+Run individual commands to interact with the local Temporal server.
+
+```bash
+temporal operator namespace list
+temporal workflow list
+```
+
+## Configuration
+
+Use the help flag to see a full list of CLI options:
+
+```bash
+temporal server start-dev --help
+```
+
+### Environmental variables
+
+The Temporal CLI hosts a set of Client Options that can be used to configure the system environment.
+Use the provided table as a reference for the Temporal CLI environmental variables.
+
+| Variable                                     | Definition                                                                        | Client Option                   |
+| -------------------------------------------- | --------------------------------------------------------------------------------- | ------------------------------- |
+| `TEMPORAL_CLI_ADDRESS`                       | The host and port (formatted as host:port) for the Temporal Frontend Service.     | --address                       |
+| `TEMPORAL_CLI_CODEC_AUTH`                    | Sets the authorization header on requests to the Codec Server.                    | --codec-auth                    |
+| `TEMPORAL_CLI_CODEC_ENDPOINT`                | Endpoint for a remote Codec Server.                                               | --codec-endpoint                |
+| `TEMPORAL_CONTEXT_TIMEOUT`                   | An optional timeout for the context of an RPC call (in seconds). Default value: 5 | --context-timeout               |
+| `TEMPORAL_CLI_NAMESPACE`                     | Identifies a Namespace in the Temporal Workflow. Default value: "default".        | --namespace                     |
+| `TEMPORAL_CLI_TLS_CA`                        | Path to server CA certificate.                                                    | --tls-ca-path                   |
+| `TEMPORAL_CLI_TLS_CERT`                      | Path to x509 certificate.                                                         | --tls-cert-path                 |
+| `TEMPORAL_CLI_TLS_DISABLE_HOST_VERIFICATION` | Disables TLS host name verification if already enabled. Default value: false.     | --tls-disable-host-verification |
+| `TEMPORAL_CLI_TLS_KEY`                       | Path to private certificate key.                                                  | --tls-key-path                  |
+| `TEMPORAL_CLI_TLS_SERVER_NAME`               | Provides an override for the target TLS server name.                              | --tls-server-name               |
+
+### Namespace registration
```

Comment:
Can we not now link to the CLI reference for this?

I don't think the README should break down every high level command usage.

This is the stuff that needs to go into the reference itself.

Having the list of "available commands" near the top will suffice.

Reply:
Actually - this makes sense in the context of running the dev Cluster.